app/views/registration/steps/_details.html.haml

```diff
@@ -57,8 +56,25 @@
                 %select.chunky{ id: 'enterprise_country', name: 'country', required: true, ng: { init: "setDefaultCountry(#{Spree::Config[:default_country_id]})", model: 'enterprise.country', options: 'c as c.name for c in countries' } }
                 %span.error{ ng: { show: "details.country.$error.required && submitted" } }
                   = t(".country_field_error")
-
+      %div{ ng: {if: "enableMapConfirm"}}
```

Comment:
Since this is a Rails view, you can use Ruby instead of Angular for this conditional.

```suggestion
      - if ENV.fetch("GOOGLE_MAPS_API_KEY")
        %div
```